### PR TITLE
Capabilities japanese translation

### PR DESCRIPTION
**Pull request by ksakae1216**
Please review for me.

Part of #586

### CODE REVIEW

File: src/content/ja/2020/capabilities.md [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+## アイドル検出API
+
+アプリケーションの中には、ユーザーがデバイスを積極的に使用しているか、アイドル状態にあるかを判断する必要があります。例えば、チャットアプリケーションは、ユーザーが不在であることを表示することがあります。画面やマウス、キーボードとのやりとりがないなど、様々な要因が考慮されます。[アイドル検出API](https://web.dev/idle-detection/) ([WICGコミュニティグループ報告書](https://wicg.github.io/idle-detection/))では、ある閾値を設定することで、ユーザがアイドル状態か画面ロック状態かを確認することができる抽象的なAPIを提供しています。
+
+これを実現するために、APIはグローバルな`window`オブジェクト上に新しい`IdleDetector`インターフェイスを提供します。開発者がこの機能を利用する前に、まず`IdleDetector.requestPermission()`を呼び出してパーミッションを要求しなければなりません。ユーザが許可を与えれば、開発者は`IdleDetector`の新しいインスタンスを作成できます。このオブジェクトは2つのプロパティを提供します。`userState`と`screenState`の2つのプロパティを提供します。このオブジェクトは、ユーザの状態と画面の状態のどちらかが変化したときに`change`イベントを発生させます。最後に、アイドル検出は`start()`メソッドを呼び出して起動する必要があります。このメソッドは2つのパラメーターを持つ設定オブジェクトを受け取ります。ユーザーがアイドル状態でなければならない時間をミリ秒単位で定義する`threshold`と、開発者はオプションで`AbortSignal`を`abort`プロパティに渡すことができます。
+
+{{ figure_markup(
+  image="idle_detection_api.png",
+  alt="アイドル検出APIを使用したChromeでのページロードの割合",
+  caption='アイドル検出APIを使用したChromeでのページロードの割合<br>(出典. <a href="https://chromestatus.com/metrics/feature/timeline/popularity/3017">アイドル検出</a>'.),
```
Comment (bazzadp) [inline src/content/ja/2020/capabilities.md:177]: This is causing it to fail to display and so pass the tests:

```suggestion
  caption='アイドル検出APIを使用したChromeでのページロードの割合<br>(出典. <a href="https://chromestatus.com/metrics/feature/timeline/popularity/3017">アイドル検出</a>.)',
```

**Review by bazzadp — CHANGES_REQUESTED** [review]
Could you fix the rendering issue and then will have a proper look later.

File: src/content/ja/2020/capabilities.md [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+
+[Capabilities Project](https://www.chromium.org/teams/web-capabilities-fugu)は、非公式にはProject Fuguとしても知られていますがGoogle、Microsoft、Intelの3社によるWebとネイティブの間のギャップを埋めるためのクロスカンパニーの取り組みです。これは、プラットフォームとしてのWebの関連性を保つために重要です。そのために、Chromiumの貢献者たちは、ユーザーのセキュリティ、プライバシー、信頼性を維持しながらOSの機能をWebに公開する新しいAPIを実装しています。これらの機能には以下のものが含まれますが、これらに限定されません。
+
+- ローカルファイルシステム上のファイルにアクセスするための[ファイルシステムアクセスAPI](https://web.dev/file-system-access/)
+- 特定のファイル拡張子のハンドラーとして登録するための[ファイルハンドラーAPI](https://web.dev/file-handling/)
+- ユーザーのクリップボードにアクセスするには、[非同期クリップボードAPI](https://web.dev/async-clipboard/)を使用します。
+- 他のアプリケーションとファイルを共有するための[Web共有API](https://web.dev/web-share/)
+- ユーザーのアドレス帳から連絡先にアクセスするには、[連絡先ピッカーAPI](https://web.dev/contact-picker/)を使用します。
+- 画像中の顔やバーコードを効率的に検出するための[形状検出API](https://web.dev/shape-detection/)
+- [Web NFC](https://web.dev/nfc/)、[Web Serial](https://web.dev/serial/)、[Web USB](https://web.dev/usb/)、[Web Bluetooth](https://web.dev/bluetooth/)、およびその他のAPI（全リストについては、[Fugu API Tracker](https://goo.gle/fugu-api-tracker)を参照）
```
Comment (bazzadp) [inline src/content/ja/2020/capabilities.md:35]:
```suggestion
- <a lang="en" href="https://web.dev/nfc/">Web NFC</a>、<a href="https://web.dev/nfc/">Web Serial</a>、<a lang="en" https://web.dev/usb/>Web USB</a>、<a leng="en" href="https://web.dev/bluetooth/">Web Bluetooth</a>、およびその他のAPI（全リストについては、<a lang="en" href="https://goo.gle/fugu-api-tracker">Fugu API Tracker</a>を参照）
```

File: src/content/ja/2020/capabilities.md [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+registration.showNotification('Title', {
+  body: 'Message',
+  showTrigger: new TimestampTrigger(timestamp),
+});
+```
+
+{{ figure_markup(
+  image="notification_triggers_api.png",
+  alt="Notification Triggers APIを使用したChromeでのページロードの割合",
+  caption='Notification Triggers APIを使用したChromeでのページ読み込みの割合<br>（出典 <a href="https://chromestatus.com/metrics/feature/timeline/popularity/3017">通知トリガー</a>）',
```
Comment (bazzadp) [inline src/content/ja/2020/capabilities.md:141]: Some hidden characters here breaking the linting:

```suggestion
  caption='Notification Triggers APIを使用したChromeでのページ読み込みの割合<br>（出典 <a href="https://chromestatus.com/metrics/feature/timeline/popularity/3017">通知トリガー</a>）',
```

**Review by bazzadp — CHANGES_REQUESTED** [review]

File: src/content/ja/2020/capabilities.md [hunk 4]
```diff
@@ -0,0 +1,328 @@
...
+featured_stat_label_1: (Chrome) Async クリップボード API を使用してページを読み込む
+featured_stat_2: 0.49%
+featured_stat_label_2: Storage Manager APIを使用しているサイト
+featured_stat_3: 363
+featured_stat_label_3: 関連アプリのインストールを許可しているサイト
+---
+
+## 序章
+
+[Progressive Web Apps](./pwa) (PWA)はWeb技術をベースにしたクロスプラットフォームのアプリケーションモデルです。サービスワーカーの協力を得て、これらのアプリケーションはユーザーがオフラインの時でも実行されます。[Webアプリマニフェスト](https://developer.mozilla.org/ja/docs/Web/Manifest)を利用すると、ホーム画面やプログラムリストにPWAを追加できます。そこから開くと、PWAはネイティブアプリケーションとして表示されます。ただしPWAはWebプラットフォームAPIを通して公開されている機能のみを使用できます。任意のネイティブインターフェイスを呼び出すことはできず、ネイティブアプリケーションとWebアプリケーションの間にギャップが残ります。
```
[the remark this answers is not in the document]
Comment (ksakae1216) [reply src/content/ja/2020/capabilities.md]: I see. This pattern fix.

File: src/content/ja/2020/capabilities.md [hunk 5]
```diff
@@ -0,0 +1,328 @@
...
+
+[Capabilities Project](https://www.chromium.org/teams/web-capabilities-fugu)は、非公式にはProject Fuguとしても知られていますがGoogle、Microsoft、Intelの3社によるWebとネイティブの間のギャップを埋めるためのクロスカンパニーの取り組みです。これは、プラットフォームとしてのWebの関連性を保つために重要です。そのために、Chromiumの貢献者たちは、ユーザーのセキュリティ、プライバシー、信頼性を維持しながらOSの機能をWebに公開する新しいAPIを実装しています。これらの機能には以下のものが含まれますが、これらに限定されません。
```
[the remark this answers is not in the document]
Comment (ksakae1216) [reply src/content/ja/2020/capabilities.md]: Thanks, I fixed.

File: src/content/ja/2020/capabilities.md (hunk 2 above)
Comment (ksakae1216) [reply src/content/ja/2020/capabilities.md]: Thanks, I fixed.

File: src/content/ja/2020/capabilities.md (hunk 3 above)
Comment (ksakae1216) [reply src/content/ja/2020/capabilities.md]: Thanks, I fixed.

**Comment by ksakae1216** [comment]
@bazzadp Help me.
I don't know why failed Test Website.
Please tell me what cause of error.

File: src/content/ja/2020/capabilities.md [hunk 6]
```diff
@@ -0,0 +1,332 @@
...
+これを実現するために、APIはグローバルな`window`オブジェクト上に新しい`IdleDetector`インターフェイスを提供します。開発者がこの機能を利用する前に、まず`IdleDetector.requestPermission()`を呼び出してパーミッションを要求しなければなりません。ユーザが許可を与えれば、開発者は`IdleDetector`の新しいインスタンスを作成できます。このオブジェクトは2つのプロパティを提供します。`userState`と`screenState`の2つのプロパティを提供します。このオブジェクトは、ユーザの状態と画面の状態のどちらかが変化したときに`change`イベントを発生させます。最後に、アイドル検出は`start()`メソッドを呼び出して起動する必要があります。このメソッドは2つのパラメーターを持つ設定オブジェクトを受け取ります。ユーザーがアイドル状態でなければならない時間をミリ秒単位で定義する`threshold`と、開発者はオプションで`AbortSignal`を`abort`プロパティに渡すことができます。
+
+{{ figure_markup(
+  image="idle_detection_api.png",
+  alt="アイドル検出APIを使用したChromeでのページロードの割合",
+<<<<<<< HEAD
+  caption='アイドル検出APIを使用したChromeでのページロードの割合<br>(出典. <a href="https://chromestatus.com/metrics/feature/timeline/popularity/3017">アイドル検出</a>。)',
+=======
+  caption='アイドル検出APIを使用したChromeでのページロードの割合<br>(出典. <a href="https://chromestatus.com/metrics/feature/timeline/popularity/3017">アイドル検出</a>.)',
+>>>>>>> 7cbbddd1d53bdd68c54abbf24c7283a5c6bf896c
```
Comment (bazzadp) [inline src/content/ja/2020/capabilities.md:181]: Looks like you have a merge conflict here @ksakae1216 so it has both lines and is waiting for you to choose which one to us:

```suggestion
  caption='アイドル検出APIを使用したChromeでのページロードの割合<br>(出典. <a href="https://chromestatus.com/metrics/feature/timeline/popularity/3017">アイドル検出</a>。)',
```

BTW if you click on the error you'll be given a clue as to why it's failing:

image

Line numbers won't match up completely as this is the generated HTML but hopefully enough context to give you a clue. In this case I searched for "idle_detection_api.png" and saw the error.

Comment (ksakae1216) [reply src/content/ja/2020/capabilities.md]: @bazzadp Thanks, I see!!
Next time, I carefully confirm log.

File: src/content/ja/2020/capabilities.md [hunk 7]
```diff
@@ -0,0 +1,328 @@
...
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTxqot9ALgxcgOVJntkzIKnkpo3idIPy-tL0t_nzC5BwFuq0ThgK5OXOYVVOpama4vB2EyggX813d33/pubchart?oid=963792757&format=interactive",
+  sheets_gid="1324588405"
+  )
+}}
+
+本稿執筆時点では、アイドル検出APIはオリジントライアル中のため、今後APIの形は変わる可能性があります。また、同様の理由から使用率は非常に低く、測定可能なものはほとんどありません。
+
+## 周期的バックグラウンド同期API
+
+ユーザーがウェブアプリケーションを閉じると、バックエンドサービスとの通信ができなくなります。いくつかのケースでは、ネイティブアプリケーションができるように、開発者は多かれ少なかれ定期的にデータを同期させたいと思うかもしれません。例えば、ニュースアプリケーションは、ユーザーが目覚める前に最新のヘッドラインをダウンロードしたいかもしれません。<a lang="en" href="https://web.dev/periodic-background-sync/">周期的バックグラウンド同期API</a>(<a lang="en" href="https://wicg.github.io/periodic-background-sync/">WICGコミュニティグループ報告書</a>)は、Webとネイティブの間のこのギャップを埋めることを目指しています。
```
Comment (bazzadp) [inline src/content/ja/2020/capabilities.md:188]:
```suggestion
ユーザーがウェブアプリケーションを閉じると、バックエンドサービスとの通信ができなくなります。いくつかのケースでは、ネイティブアプリケーションができるように、開発者は多かれ少なかれ定期的にデータを同期させたいと思うかもしれません。例えば、ニュースアプリケーションは、ユーザーが目覚める前に最新のヘッドラインをダウンロードしたいかもしれません。[周期的バックグラウンド同期API](https://web.dev/periodic-background-sync/)([WICGコミュニティグループ報告書](https://wicg.github.io/periodic-background-sync/)は、Webとネイティブの間のこのギャップを埋めることを目指しています。
```

File: src/content/ja/2020/capabilities.md [hunk 8]
```diff
@@ -0,0 +1,328 @@
...
+
+```js
+self.addEventListener('periodicsync', (event) => {
+  if (event.tag === 'articles') {
+    event.waitUntil(syncStuff());
+  }
+});
+```
+
+この記事を書いている時点で、このAPIを実装しているのはChromiumベースのブラウザのみです。これらのブラウザでは、APIを使用する前にアプリケーションをインストール（ホーム画面に追加）する必要があります。ウェブサイトの<a lang="en" href="https://www.chromium.org/developers/design-documents/site-engagement">サイトエンゲージメントスコア</a> は、定期的な同期イベントを呼び出せるか、どのくらいの頻度で呼び出せるかを定義します。現在の保守的な実装では、ウェブサイトは1日1回コンテンツを同期できます。
```
Comment (bazzadp) [inline src/content/ja/2020/capabilities.md:212]:
```suggestion
この記事を書いている時点で、このAPIを実装しているのはChromiumベースのブラウザのみです。これらのブラウザでは、APIを使用する前にアプリケーションをインストール（ホーム画面に追加）する必要があります。ウェブサイトの[サイトエンゲージメントスコア](https://www.chromium.org/developers/design-documents/site-engagement) は、定期的な同期イベントを呼び出せるか、どのくらいの頻度で呼び出せるかを定義します。現在の保守的な実装では、ウェブサイトは1日1回コンテンツを同期できます。
```

File: src/content/ja/2020/capabilities.md [hunk 9]
```diff
@@ -0,0 +1,328 @@
...
+  )
+}}
+
+コンテンツインデックスAPIは、2020年7月にChrome 84で提供が開始されました。提供直後は、Chromeでのページロードの約0.0002%の間にAPIが使用されていました。2020年10月時点では、この値はほぼ10倍に増加しています。
+
+## 新しいトランスポートAPI
+
+最後に、現在オリジントライアル中の2つの新しいトランスポート方式があります。1つ目は開発者がWebSocketsで高頻度のメッセージを受信できるようにするもので、2つ目はHTTPやWebSocketsとは別に全く新しい双方向通信プロトコルを導入しています。
+
+### WebSocketsのBackpressure
```
Comment (bazzadp) [inline src/content/ja/2020/capabilities.md:273]: Titles need special handling so the span tags are not included in the generated link:

```suggestion
### <span lang="en">WebSockets</span>の<span lang="en">Backpressure</span> {websocketsのbackpressure}
```

File: src/content/ja/2020/capabilities.md [hunk 10]
```diff
@@ -0,0 +1,328 @@
...
+
+WebSocket APIは、Webサイトとサーバー間の双方向通信に最適です。しかし、WebSocket APIはbackpressureを許さないので、高頻度のメッセージを扱うアプリケーションはフリーズする可能性があります。<a lang="en" href="https://web.dev/websocketstream/">WebSocketStream API</a>（<a lang="en" href="https://github.com/ricea/websocketstream-explainer/blob/master/README.md">の説明者</a>は、まだ標準化トラックには乗っていません）は、WebSocket APIをストリームで拡張することは、使いやすいbackpressureのサポートをWebSocket APIにもたらしたいと考えています。通常の`WebSocket`コンストラクタを使う代わりに、開発者は`WebSocketStream`インターフェイスの新しいインスタンスを作成する必要があります。ストリームの`connection`プロパティは、読み込みと書き込み可能なストリームへ解決する約束を返します。
```
Comment (bazzadp) [inline src/content/ja/2020/capabilities.md:275]:
```suggestion
<span lang="en">WebSocket API</span>は、Webサイトとサーバー間の双方向通信に最適です。しかし、<span lang="en">WebSocket API</span>は<span lang="en">backpressure</span>を許さないので、高頻度のメッセージを扱うアプリケーションはフリーズする可能性があります。<a lang="en" href="https://web.dev/websocketstream/">WebSocketStream API</a>（[の説明者](https://github.com/ricea/websocketstream-explainer/blob/master/README.md)は、まだ標準化トラックには乗っていません）は、<span lang="en">WebSocket API</span>をストリームで拡張することは、使いやすい<span lang="en">backpressure</span>のサポートを<span lang="en">WebSocket API</span>にもたらしたいと考えています。通常の`WebSocket`コンストラクタを使う代わりに、開発者は`WebSocketStream`インターフェイスの新しいインスタンスを作成する必要があります。ストリームの`connection`プロパティは、読み込みと書き込み可能なストリームへ解決する約束を返します。
```

File: src/content/ja/2020/capabilities.md [hunk 11]
```diff
@@ -0,0 +1,328 @@
...
+  image="websocketstreams.png",
+  alt="WebSocketStreamsを使用したChromeでのページロードの割合",
+  caption='WebSocketStreamsを使用したChromeでのページ読み込みの割合<br>（出典 <a href="https://chromestatus.com/metrics/feature/timeline/popularity/3018">WebSocketStream</a>)',
+  description="この機能を使用してChromeでのページロードの割合に基づくWebSocketStreamsの使用状況のグラフです。2020年6月と7月にピークを迎え、約0.0008%のページロード中にAPIが使用されたことを示しています。",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTxqot9ALgxcgOVJntkzIKnkpo3idIPy-tL0t_nzC5BwFuq0ThgK5OXOYVVOpama4vB2EyggX813d33/pubchart?oid=1714443590&format=interactive",
+  sheets_gid="691106754"
+  )
+}}
+
+WebSocketStream APIは最初のオリジントライアルを終え、再び実験段階に戻っています。これは、現在このAPIの使用率が非常に低く、ほとんど測定できない理由も説明しています。
```
Comment (bazzadp) [inline src/content/ja/2020/capabilities.md:296]:
```suggestion
<span lang="en">WebSocketStream API</span>は最初のオリジントライアルを終え、再び実験段階に戻っています。これは、現在このAPIの使用率が非常に低く、ほとんど測定できない理由も説明しています。
```

File: src/content/ja/2020/capabilities.md [hunk 12]
```diff
@@ -0,0 +1,328 @@
...
+
+### 速くする
+
+<a lang="en" href="https://www.chromium.org/quic">QUIC</a>(<a lang="en" href="https://www.ietf.org/archive/id/draft-ietf-quic-transport-31.txt">IETFインターネット草案</a>)は、UDP上に実装された多重化されたストリームベースの双方向トランスポートプロトコルです。これは、TCP上に実装されているHTTP/WebSocket APIに代わるものです。<a lang="en" href="https://web.dev/quictransport/">QuicTransport API</a>は、QUICサーバとメッセージを送受信するためのクライアント側APIです。開発者は、データグラムを介して信頼性のないデータを送信するか、そのストリームAPIを使用して信頼性の高いデータを送信するかを選択できます。
```
Comment (bazzadp) [inline src/content/ja/2020/capabilities.md:300]:
```suggestion
[QUIC](https://www.chromium.org/quic)([IETFインターネット草案](https://www.ietf.org/archive/id/draft-ietf-quic-transport-31.txt))は、UDP上に実装された多重化されたストリームベースの双方向トランスポートプロトコルです。これは、TCP上に実装されている<span lang="en">HTTP/WebSocket API</span>に代わるものです。<a lang="en" href="https://web.dev/quictransport/">QuicTransport API</a>は、QUICサーバとメッセージを送受信するためのクライアント側APIです。開発者は、データグラムを介して信頼性のないデータを送信するか、そのストリームAPIを使用して信頼性の高いデータを送信するかを選択できます。
```

File: src/content/ja/2020/capabilities.md [hunk 13]
```diff
@@ -0,0 +1,328 @@
...
+  )
+}}
+
+現在はまだほとんど測定できないほどの使用率です。2020年10月には強力に増加し、現在ではChromeでのページロードの0.00089%の間に使用されています。
+
+## 結論
+
+2020年のWeb機能の状態は健全であり、Chromiumベースのブラウザの新しいリリースには、新しくて強力なAPIが定期的に提供されています。コンテンツインデックスAPIやアイドル検出APIのようないくつかのインターフェイスは、特定のWebアプリケーションに仕上げの機能を追加するのに役立ちます。ファイルシステムアクセスや非同期クリップボードAPIのような他のAPIは、生産性アプリケーションという全く新しいアプリケーションカテゴリをWebに完全に移行させることを可能にします。非同期クリップボードやWeb共有APIのようなAPIの一部は、すでに他の非Chromiumブラウザにも導入されています。Safariは、Web共有APIを実装した最初のモバイルブラウザでした。
+
+ふぐチームは、<a lang="en" href="https://developers.google.com/web/updates/capabilities#process">厳格なプロセス</a>を通じて、これらの機能へのアクセスが安全かつプライバシーに配慮した方法で行われることを保証します。さらに、ふぐチームは他のブラウザベンダーやウェブ開発者からの<a lang="en" href="mailto:fugu-dev@chromium.org">フィードバック</a>を積極的に募集しています。これらの新しいAPIのほとんどの使用率は比較的低いですが、本章で紹介するAPIの中にはバッジAPIやコンテンツインデックスAPIのように指数関数的、あるいはホッケーの棒のように成長しているものもあります。2021年のウェブ機能のあり方は、ウェブ開発者自身にかかっています。著者は素晴らしいWebアプリケーションを構築し、強力なAPIを後方互換性のある方法で活用して、Webをより有能なプラットフォームにするための手助けをすることをコミュニティに奨励しています。
```
Comment (bazzadp) [inline src/content/ja/2020/capabilities.md:328]:
```suggestion
ふぐチームは、[厳格なプロセス](https://developers.google.com/web/updates/capabilities#process)を通じて、これらの機能へのアクセスが安全かつプライバシーに配慮した方法で行われることを保証します。さらに、ふぐチームは他のブラウザベンダーやウェブ開発者からの[フィードバック](mailto:fugu-dev@chromium.org)を積極的に募集しています。これらの新しいAPIのほとんどの使用率は比較的低いですが、本章で紹介するAPIの中にはバッジAPIやコンテンツインデックスAPIのように指数関数的、あるいはホッケーの棒のように成長しているものもあります。2021年のウェブ機能のあり方は、ウェブ開発者自身にかかっています。著者は素晴らしいWebアプリケーションを構築し、強力なAPIを後方互換性のある方法で活用して、Webをより有能なプラットフォームにするための手助けをすることをコミュニティに奨励しています。
```

**Review by bazzadp — CHANGES_REQUESTED** [review]
I think I may have confused you @ksakae1216 

The `lang="en"` is used when the enclosing text is in English. It is used in screenreaders to tell it to read this bit of text in English. It is not used as an indicator that the link is to an English document.

So in a lot of cases where you added `lang="en"` to a link, you shouldn't have, as you had translated the link text so it was in Japanese and not in English.

So I have reverted those cases back. I also added some more that I think should be in there.

Does that make sense?

File: src/content/ja/2020/capabilities.md [hunk 14]
```diff
@@ -0,0 +1,328 @@
...
+- 他のアプリケーションとファイルを共有するための<a lang="en" href="https://web.dev/web-share/">Web共有API</a>。
+- ユーザーのアドレス帳から連絡先にアクセスするには、<a lang="en" href="https://web.dev/contact-picker/">連絡先ピッカーAPI</a>を使用します。
+- 画像中の顔やバーコードを効率的に検出するための<a lang="en" href="https://web.dev/shape-detection/">形状検出API</a>。
+- <a lang="en" href="https://web.dev/nfc/">Web NFC</a>、<a lang="en" href="https://web.dev/serial/">Web Serial</a>、<a lang="en" href="https://web.dev/usb/">Web USB</a>、<a lang="en" href="https://web.dev/bluetooth/">Web Bluetooth</a>、およびその他のAPI（全リストについては、<a lang="en" href="https://goo.gle/fugu-api-tracker">Fugu API Tracker</a>を参照）
+
+誰でも新しい機能を提案するには、<a lang="en" href="https://bit.ly/new-fugu-request">Chromiumバグトラッカーでチケットを作成</a>する必要があります。Chromiumのコントリビューターは提案を検討し、適切な標準化団体を通じて他の開発者やブラウザベンダーとすべてのAPIについて議論します。一方、ふぐチームはChromiumでAPIを実装します。その後、APIは<a lang="en" href="https://web.dev/origin-trials/">オリジントライアル</a>を通じて限られた人たちへ利用可能になります。この段階では開発者は特定のオリジンでAPIをテストするためのトークンにサインアップできます。APIが十分に堅牢であることが判明した場合、APIはChromiumで提供され、ベンダーが決定した場合は他のブラウザでも提供されます。<a lang="en" href="https://web.dev/fugu-status/">Capability Status</a>のサイトでは、異なるCapability APIがどこで行われているかを示しています。
```
[the remark this answers is not in the document]
Comment (ksakae1216) [reply src/content/ja/2020/capabilities.md:37]: Thanks, I fixed.
And line 29-34 too.

File: src/content/ja/2020/capabilities.md [hunk 15]
```diff
@@ -0,0 +1,328 @@
...
+
+ふぐプロジェクト、Capabilitiesプロジェクトのコードネームであるふぐは、日本料理にちなんで名付けられたもので、正しく調理されたふぐの肉は特別な味を体験できる。しかし正しく調理されていない場合は、致命的になる可能性があります。ふぐプロジェクトの強力なAPIは、開発者にとって非常にエキサイティングなものです。ユーザーのセキュリティやプライバシーに影響を与える可能性があります。そのため、ふぐチームはこれらの問題に特別な注意を払っています。例えば新しいインターフェイスでは、Webサイトを安全な接続（HTTPS）で送信する必要があります。中には、不正行為を防ぐために、クリックやキーを押すといったユーザーのジェスチャーを必要とするものもあります。その他の機能では、ユーザーによる明示的な許可が必要となります。開発者は、すべてのAPIを段階的な強化として使用できます。APIを機能検出することで、これらの機能をサポートしていないブラウザでアプリケーションが壊れることはありません。APIをサポートしているブラウザでは、ユーザーはより良い体験を得ることができます。このようにしてWebアプリはユーザーの特定のブラウザに応じて<a lang="en" href="https://web.dev/progressively-enhance-your-pwa/">段階的に強化</a>します。
```
[the remark this answers is not in the document]
Comment (ksakae1216) [reply src/content/ja/2020/capabilities.md:39]: Thanks, I fixed.

File: src/content/ja/2020/capabilities.md [hunk 16]
```diff
@@ -0,0 +1,328 @@
...
+
+本章では、HTTP Archiveと<a lang="en" href="https://chromestatus.com/metrics/feature/timeline/popularity">Chrome Platform Status</a>による利用状況データをもとに、様々な最新のWeb APIの概要と2020年のWeb機能の状況を紹介します。一部のインターフェイスは真新しいものもあるため、（相対的な）利用率は非常に低いです。そのため、他の章とは異なりHTTP Archiveの利用統計は相対的な割合ではなく、絶対的なページ数で表示されます。<a lang="en" href="./methodology#metrics">技術的な制限</a>のため、HTTP Archiveには、許可もユーザーのジェスチャーも必要としないAPIのデータしかありません。データがない場合は、代わりにChrome Platform Statusに従ったGoogle Chromeでのページロードのパーセンテージが表示されます。統計が必ずしも意味のあるものではないほど数値が小さくても、多くの場合はデータから傾向を読み取ることができます。また、これらの統計値は、この章の今後の年次版で、APIがどの程度成熟し、採用率が向上したかを振り返る際のベースラインとしても使用できます。特に断りのない限り、APIはChromiumベースのブラウザでのみ利用可能であり、その仕様は標準化の初期段階にあります。
```
[the remark this answers is not in the document]
Comment (ksakae1216) [reply src/content/ja/2020/capabilities.md:41]: Thanks, I fixed.

File: src/content/ja/2020/capabilities.md [hunk 17]
```diff
@@ -0,0 +1,328 @@
...
+
+## 非同期クリップボードAPI
+
+`document.execCommand()`メソッドの助けを借りて、Webサイトはすでにユーザのクリップボードにアクセスできました。しかしAPIは同期的であり（クリップボードの項目を処理するのが難しい）、DOM内の選択されたテキストとしか対話できないため、このアプローチには多少の制限があります。そこで登場するのが <a lang="en" href="https://webkit.org/blog/10855/async-clipboard-api/">非同期クリップボードAPI</a>(<a lang="en" href="https://www.w3.org/TR/clipboard-apis/#async-clipboard-api">W3C草案</a>です。この新しいAPIは非同期であるだけでなく、大きなデータの塊のためにページをブロックしたり、許可が下りるのを待ったりしないという意味でもあります。
```
[the remark this answers is not in the document]
Comment (ksakae1216) [reply src/content/ja/2020/capabilities.md:45]: Thanks, I fixed.

File: src/content/ja/2020/capabilities.md [hunk 18]
```diff
@@ -0,0 +1,328 @@
...
+  )
+}}
+
+非同期クリップボードAPIは比較的新しいAPIなので、現在のところ利用率はかなり低いです。2020年3月、SafariはSafari 13.1で非同期クリップボードAPIのサポートを追加しました。2020年の間に、`read()` APIの使用量は増加しました。2020年10月には、Google Chromeの全ページロードの0.0003%の間にAPIが呼び出されました。
+
+### 書込みアクセス
+
+読み込み操作とは別に、非同期クリップボードAPIにはクリップボードに内容を書き込むための2つのメソッドがあります。ここでもプレーンテキスト用の短縮メソッド `navigator.clipboard.writeText()` と、任意のデータ用のメソッド `navigator.clipboard.write()` があります。Chromiumベースのブラウザでは、タブがアクティブな状態でクリップボードに書き込むことは許可を必要としません。しかし、Webサイトがバックグラウンドにある時、クリップボードに書き込もうとすると許可が必要になります。この方法はユーザのジェスチャーとパーミッションが必要なので、HTTP Archiveのメトリクスの対象外となります。`read()` メソッドとは対照的に、`write()` メソッドの使用率は指数関数的に増加しており、2020年10月には全ページロードの0.0006%の一部になっています。
+
+ふぐのもう1つの機能である<a lang="en" href="https://bugs.chromium.org/p/chromium/issues/detail?id=897289">RawクリップボードアクセスAPI</a>は、任意のデータをクリップボードからコピーしたり、クリップボードに貼り付けたりできるようにすることで非同期クリップボードAPIをさらに強化できます。
```
[the remark this answers is not in the document]
Comment (ksakae1216) [reply src/content/ja/2020/capabilities.md:67]: Thanks, I fixed.

File: src/content/ja/2020/capabilities.md (hunk 9 above)
Comment (ksakae1216) [reply src/content/ja/2020/capabilities.md:273]: Thanks, I fixed.

File: src/content/ja/2020/capabilities.md (hunk 10 above)
Comment (ksakae1216) [reply src/content/ja/2020/capabilities.md:275]: Thanks, I fixed.

File: src/content/ja/2020/capabilities.md (hunk 11 above)
Comment (ksakae1216) [reply src/content/ja/2020/capabilities.md:296]: Thanks, I fixed.

File: src/content/ja/2020/capabilities.md (hunk 12 above)
Comment (ksakae1216) [reply src/content/ja/2020/capabilities.md:300]: Thanks, I fixed.

File: src/content/ja/2020/capabilities.md (hunk 13 above)
Comment (ksakae1216) [reply src/content/ja/2020/capabilities.md:328]: Thanks, I fixed.

File: src/content/ja/2020/capabilities.md [hunk 19]
```diff
@@ -0,0 +1,331 @@
...
+    event.waitUntil(syncStuff());
+  }
+});
+```
+
+この記事を書いている時点で、このAPIを実装しているのはChromiumベースのブラウザのみです。これらのブラウザでは、APIを使用する前にアプリケーションをインストール（ホーム画面に追加）する必要があります。ウェブサイトの[サイトエンゲージメントスコア](https://www.chromium.org/developers/design-documents/site-engagement)は、定期的な同期イベントを呼び出せるか、どのくらいの頻度で呼び出せるかを定義します。現在の保守的な実装では、ウェブサイトは1日1回コンテンツを同期できます。
+
+{{ figure_markup(
+  image="periodic_background_sync_api.png",
+  caption="周期的バックグラウンド同期APIを使用しているページ数。",
```
Comment (bazzadp) [inline src/content/ja/2020/capabilities.md:219]:
```suggestion
  alt="周期的バックグラウンド同期APIを使用しているページ数。",
  caption="周期的バックグラウンド同期APIを使用しているページ数。",
```

File: src/content/ja/2020/capabilities.md [hunk 20]
```diff
@@ -0,0 +1,332 @@
...
+Storage Manager APIは2016年からChrome、2017年からFirefox、そして新しいChromiumベースのEdgeでサポートされています。HTTP Archiveのデータによると、デスクトップページ27,056ページ（0.49%）とモバイルページ34,042ページ（0.48%）でAPIが使用されています。2020年の間に、Storage Manager APIの使用率は増加の一途をたどっています。このことからも、この章では、このインターフェイスが最もよく使われているAPIとなっています。
+
+### 永続的ストレージへのオプトイン
+
+ウェブストレージには2つのカテゴリーがあります。「ベストエフォート」と「パーシステント」の2種類があり、前者がデフォルトとなっています。デバイスのストレージが少なくなると、ブラウザは自動的にベストエフォートストレージを解放しようとします。例えば、FirefoxとChromiumベースのブラウザは、最近使用されていないものからストレージを削除します。しかし、これは重要なデータを失うリスクがあります。退避を防ぐために、開発者は永続的なストレージを選ぶことができます。この場合、容量が少なくなってもブラウザはストレージをクリアしません。ユーザーは手動でストレージをクリアすることもできます。持続的ストレージを選択するには、開発者は`navigator.storage.persist()`メソッドを呼び出す必要があります。ブラウザとサイトの利用状況によっては、許可のプロンプトが表示されたり、リクエストが自動的に受け入れられたり拒否されたりすることがあります。
+
+{{ figure_markup(
+  image="storage_manager_api_persist.png",
+  alt="StorageManager APIのpersistメソッドの使用ページ数。",
+  caption='<span lang="en">StorageManager API</span>の<span lang="en>persist</span>メソッドの使用ページ数。',
```
Comment (bazzadp) [inline src/content/ja/2020/capabilities.md:99]:
```suggestion
  caption='<span lang="en">StorageManager API</span>の<span lang="en">persist</span>メソッドの使用ページ数。',
```

**Review by bazzadp — APPROVED** [review]

**Pull request merged**